docs/operators/droplet-quickstart.md

```diff
@@ -0,0 +1,240 @@
+
+Choose the Droplet size that you are most comfortable with, and then run the following command -
+
+```bash
+export DROPLET_SIZE=<slug>
+```
+
+For example, `s-1vcpu-2gb` is more than capable to handle the protocols we mentioned above -
+
+```bash
+export DROPLET_SIZE=s-1vcpu-2gb
+```
+
+## 6. Create the Droplet
+
+Run the following command to create the droplet -
+
+```bash
+export DROPLET_NAME=<your-droplet-name>
+export DROPLET_ID=$(doctl compute droplet create --region=$DROPLET_REGION --image=$DROPLET_IMAGE --size=$DROPLET_SIZE --enable-monitoring --format=ID --wait $DROPLET_NAME | sed -n '2 p')
+```
+
+For example, to create a droplet named `nwaku` -
+
+```bash
+export DROPLET_NAME=nwaku
+export DROPLET_ID=$(doctl compute droplet create --region=$DROPLET_REGION --image=$DROPLET_IMAGE --size=$DROPLET_SIZE --enable-monitoring --format=ID --wait $DROPLET_NAME | sed -n '2 p')
+```
+
+## 7. Create a Domain and attach it to the droplet
+
+Follow this [guide](https://docs.digitalocean.com/products/networking/dns/how-to/add-domains/) to create a domain, and add it to the droplet appropriately.
+
+## 8. SSH into the Droplet
+
+You can get the following details in the email that DigitalOcean sends upon successful creation of the Droplet -
+
+1. username
+2. password
+3. public ipv4 address
+
+Since the public key we previously generated was automatically added to the authorized_keys list, we can run the following command to ssh into the Droplet -
+
+```bash
+export DROPLET_USERNAME=<username from email>
+export DROPLET_IP=<public ipv4 address from email>
+ssh -i $DROPLET_SSH_KEY_PATH $DROPLET_USERNAME@$DROPLET_IP
+```
+
+For example, if the username was `root`, and the ipv4 address was `0.0.0.0`,
+
+```bash
+export USERNAME=root
+export IP=0.0.0.0
+ssh -i $DROPLET_SSH_KEY_PATH $DROPLET_USERNAME@$DROPLET_IP
+```
+
+Enter the password received in the email.
+
+## 9. Build nwaku
+
+To build `nwaku`, follow this [guide](./how-to/build.md)
+
+OR
+
+To fetch the latest release from Github, navigate to https://github.com/status-im/nwaku/releases and download the latest tarball for your distribution.
+
+This [guide](https://www.itprotoday.com/development-techniques-and-management/how-install-targz-file-ubuntu-linux) describes how to install a tarball for your distribution.
+
+OR
+
+Run the following script to copy over the wakunode2 binary (from the host machine) -
+
+```bash
+scp -i $DROPLET_SSH_KEY_PATH ./wakunode2 $USERNAME@$IP:~/wakunode2
+```
+
+## 10. Run nwaku
+
+Run the following command to run `nwaku` -
+
+*Note the path to the wakunode2 binary*
+
+```bash
+export DOMAIN_NAME=<your-domain-name>
+./build/wakunode2 \
+  --store:true \
+  --storenode:/dns4/node-01.ac-cn-hongkong-c.wakuv2.test.statusim.net/tcp/30303/p2p/16Uiu2HAkvWiyFsgRhuJEb9JfjYxEkoHLgnUQmr1N5mKWnYjxYRVm \
```

Comment:
I'm not sure if it's worth enabling `store` in these first setup instructions. For one, to be a `store` server, it would also require the `--persist-messages` parameter, but would not be used as a store node by other nodes until we have properly configured capability discovery. Setting a `--storenode` is also unnecessary, as this sets up the node as a store client, which is typically only necessary for client applications (it also activates ft-store and "resuming" from the configured `storenode` - which has not been tested thoroughly). WDYT? In order to keep the first operator trial as clean as possible, perhaps we simply focus on setting up nodes for `relay`?

Reply:
Sure, we can remove the storenode option. It would be good to have operators run nodes with store enabled so we can have a larger sample size for the optimisations we make

Reply:
Indeed, but in order to actually serve store messages, they would need to use `--persist-messages` too (although this is silly and we're working towards simplifying store config options)

Reply:
Addressed in [3febd72](https://github.com/status-im/nwaku/pull/1133/commits/3febd72c0923543883064290187a999d3f291c4e)